ansible/roles/vmware-vm/tasks/deploy.yml

```diff
@@ -7,7 +7,7 @@
       - "--user '{{ vcenter_username }}'"
       - "--password '{{ vcenter_password }}'"
       - "--datacenter '{{ vcenter_datacenter }}'"
-      - "--datastore-cluster '{{ vcenter_datastore_cluster }}'"
+      - "--cluster '{{ vcenter_cluster }}'"
```

Comment:
I think it would be better to choose datastores from given datastore cluster, instead of the whole cluster.